README.md

```diff
@@ -4678,4 +4678,14 @@ your computational endeavors!
 
 😸
 
+## Container Usage
+
+Copy `just` into your multistage build (automatically selects correct architecture):
+
+```dockerfile
+FROM ghcr.io/casey/just:latest as just
```

Comment:
This can be further simplified by the removal of the aliasing. It's not a requirement for a working implementation.

Reply:
This was suggested earlier but dismissed for some reason: https://github.com/casey/just/pull/2851#discussion_r2253628411

Reply:
I absolutely meant to make the change, it was late, it got lost when i flipped from github.com to my editor.

Reply:
this was an oversight on my part. i agree with @polarathene and everybody, it was really late.